portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,46 +31,95 @@
 
 
 
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
   private ArrayList<String> messages = new ArrayList<String>();
 
+  static class Data{
+     Long id;
+     String comment;
+     long timestamp;
+
+     /**Constructor*/
```

Comment:
https://stackoverflow.com/questions/34482472/how-to-write-api-documentation-for-constructors-in-java